server/lib/genome/names.rb

```diff
@@ -0,0 +1,120 @@
+module Genome
```

Comment:
@acoffman this seemed like a straightforward way to get something like a shared set of enums (rather than ruby symbols) to standardize values across importers/groupers/normalizers. Is this sufficiently idiomatic? Any suggestions?

Reply:
You can add 
`# frozen_string_literal: true` 

As the first line in the file rather than `.freeze` everywhere for this sort of thing if you prefer. You _could_ potentially make these dictionaries/hashes, but based on the usage elsewhere in this PR I think I agree that consts are actually better here!